src/cargo/core/package_id.rs

```diff
@@ -251,43 +251,6 @@ mod tests {
         assert!(PackageId::new("foo", "", repo).is_err());
     }
 
-    #[test]
```

Comment:
Yes. Due to 23e91c3d8416ce9d2099d3bc702c0ca51f5d418d, `SourceId::display_registry_name` outputs full URL instead of `crates-io` because the test used `SourceId::for_registry`. 

However, but I don't think we guarantee a stable `Debug` output so it makes less sense to keep the test there.